--- versions/4.22.0/content/themes/casper/post.hbs
+++ versions/4.22.0/content/themes/casper/post.hbs
@@ -73,11 +73,19 @@
             {{content}}
         </section>
 
-        {{!--
         <section class="article-comments gh-canvas">
-            If you want to embed comments, this is a good place to paste your code!
+        <div id="disqus_thread"></div>
+        <script>
+
+        (function() { // DON'T EDIT BELOW THIS LINE
+        var d = document, s = d.createElement('script');
+        s.src = 'https://karlstoney.disqus.com/embed.js';
+        s.setAttribute('data-timestamp', +new Date());
+        (d.head || d.body).appendChild(s);
+        })();
+        </script>
+        <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
         </section>
-        --}}
 
     </article>
 </main>
@@ -123,4 +131,4 @@
 </aside>
 {{/if}}
 
-{{/post}}
\ No newline at end of file
+{{/post}}
